app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -210,7 +215,7 @@ class QuestionPlayerActivityTest {
   }
 
   // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
-  @RunOn(TestPlatform.ESPRESSO)
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly pass on Robolectric and shouldn't be re-enabled
```

Comment:
Typo: ``Incorrectly passes on Robolectric...``

Ditto below.

Reply:
Ah, looks like I need Grammarly in Android Studio. 
Done